package.json

```
@@ -4,7 +4,8 @@
   "description": "",
   "main": "main.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "check": "npm run eslint && npm run prettier"
```

```suggestion
    "check": "eslint . && prettier --check ."
```

`npm run check`를 직접 실행해보면 에러가 날 거에요. 정상적으로 실행되게 하려면 이렇게 호출해야 합니다.

---

[ESLint CLI 레퍼런스 문서](https://eslint.org/docs/latest/use/command-line-interface)
[Prettier CLI 레퍼런스 문서](https://prettier.io/docs/en/cli.html)
를 참고하시면 좋아요